trailblazer/server/api.py

```diff
@@ -85,3 +95,122 @@ def aggregate_jobs():
 
     data = store.aggregate_failed(one_month_ago)
     return jsonify(jobs=data)
+
+
+# CG REST INTERFACE ###
+
+
+@blueprint.route("/query-analyses", methods=["POST"])
```

Comment:
I think this method should bet a GET. POST if for when a resource is added to the DB. GET when data is retrieved